### DISCUSSION TITLE
timeline

### DISCUSSION THREAD

**Discussion by carrie13** [jspsych/jsPsych#581] [Q&A]
Hey, not really sure if anyone can help with this, but I'm struggling with the timeline for this experiment. I had it running fine with the singular "image_trial" variable, but after adding more variables it no longer displays images
![screenshot 2018-12-29 12 20 19](https://user-images.githubusercontent.com/45665866/50540665-25c7d880-0b64-11e9-962a-1b635cc1975d.png)
![screenshot 2018-12-29 12 20 07](https://user-images.githubusercontent.com/45665866/50540666-2bbdb980-0b64-11e9-83e1-0cde3787bdc4.png)

**Comment by jodeleeuw** [comment]
I don't see anything wrong with the code at first glance. Are you getting any error messages in the Console when you open the file?

**Comment by jodeleeuw** [comment]
My best guess is that one of the paths to the images is wrong, or the image file is missing. That would cause jsPsych to fail to load the image file in the preloading phase and the experiment not to run.

**Comment by carrie13** [comment]
I checked it again, and the image path was the problem! Thanks!
I'll eventually need to loop this timeline, but I'm struggling to do that
with loop function... Is there an easy way to do that?

**Comment by jodeleeuw** [comment]
Depending on what you want to do you may find using [timeline variables](https://www.jspsych.org/overview/timeline/#timeline-variables) easier than looping.
